sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -133,54 +147,77 @@ export class ConfirmationOfEnrollmentController {
    * @returns application details for COE
    */
   @HasLocationAccess("locationId")
-  @Get(":locationId/confirmation-of-enrollment/application/:applicationId")
+  @Get(
+    ":locationId/confirmation-of-enrollment/disbursement/:disbursementScheduleId",
+  )
   async getApplicationForCOE(
```

Comment:
Does the name of the method still make sense? I believe that it should mention something about the disbursement now.

Reply:
This API is used to display the COE Form which still displays most of the application, program study period information. So I did not change the name.